Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.11-buster as base
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    gcc \
+    libc-dev \
+    curl \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:${PATH}"
+
+WORKDIR /todo_app
+COPY pyproject.toml poetry.toml poetry.lock ./
+RUN poetry install --no-root
+
+EXPOSE 5000
+
+FROM base as production
+ENV FLASK_DEBUG=false
+COPY . /todo_app
+ENTRYPOINT poetry run gunicorn -b 0.0.0.0:5000 "todo_app.app:create_app()"
+
+FROM base as development
+ENV FLASK_DEBUG=true
+COPY . /todo_app
+ENTRYPOINT poetry run gunicorn -b 0.0.0.0:5000 "todo_app.app:create_app()"
```

Comment:
I'd actually recommend that this stick with Flask, so that we can have the advantages of e.g. hot reloads in dev (if I remember rightly, someone may have convinced me that this can work in Gunicorn on Mac, but it won't for me!)

(Also note that Gunicorn isn't listed as a project dependency, I suspect something to do with avoiding M5 building directly M4)

Reply:
Admittedly hot reloads using Gunicorn were working just fine for me on Windows, so I didn't think to consider if it would work elsewhere. Good to know!
Gunicorn not being here is because it was added in M4, yeah.